### PR TITLE
nostri.chat

### PR DESCRIPTION
**Pull request by elidyweaver**
Added chat widget built by @pablof7z for #nostrica & #nostrica2023 
https://github.com/pablof7z/nostr-chat-widget

### CODE REVIEW

File: discover/index.html [hunk 1]
```diff
@@ -124,5 +124,19 @@ <h3 class="npub w3-large">follow the journey on our nostr:</h3>
...
 </script>
+<!--- nostri.chat widget by pablof7z--->
+<script
+  src="https://nostri.chat/public/bundle.js"
+  <!-- YOUR PUBKEY IN HEX FORMAT -->
+  data-website-owner-pubkey="YOUR_PUBKEY"
+  <!-- THE TYPE OF CHAT YOU WANT: GLOBAL or DMs -->
+  data-chat-type="GLOBAL"
+  <!-- If you use GLOBAL you can choose set a comma-separated list of hashtags-->
+  data-chat-tags="#nostrica,#nostrica2023"
```
Comment (pablof7z) [inline discover/index.html:135]: better do it without the hashtag, that was an error on my sample code I need to correct:

```
  data-chat-tags="nostrica,nostrica2023"
```

File: discover/index.html [hunk 2]
```diff
@@ -124,5 +124,19 @@ <h3 class="npub w3-large">follow the journey on our nostr:</h3>
   }
 }
 </script>
+<!--- nostri.chat widget by pablof7z--->
+<script
+  src="https://nostri.chat/public/bundle.js"
+  <!-- YOUR PUBKEY IN HEX FORMAT -->
+  data-website-owner-pubkey="YOUR_PUBKEY"
```
Comment (pablof7z) [inline discover/index.html:131]: for `chat-type="GLOBAL"` it would make sense to remove this line

**Comment by pablof7z** [comment]
so cool seeing this live! 🧡🧡🧡🧡

File: discover/index.html (hunk 2 above)
Comment (elidyweaver) [reply discover/index.html:131]: Thank you so much for reviewing the nostri.chat code. Would having the PUBKEY there, prevent anyone from writing, except for that npub?

File: discover/index.html (hunk 1 above)
Comment (elidyweaver) [reply discover/index.html:135]: Thank you, I'll remove the hashtags. I will test whether we can see the feed.

Comment (elidyweaver) [reply discover/index.html:135]: Also, does it matter which relays or how many are added to the list? I left your list intact, except for nostrica's.

**Review by elidyweaver — COMMENTED** [review]
Thank you @pablof7z. I'm eager to see how it works now.

**Comment by pablof7z** [comment]
@elidyweaver:
>  Thank you, I'll remove the hashtags. I will test whether we can see the feed.
Perfect! I sent another PR to remove the comments since HTML doesn't like the comments in between a <script> tag

>  Also, does it matter which relays or how many are added to the list? I left your list intact, except for nostrica's.
no, those should do it; it's just like typical relay selection -- this selection should be good

**Pull request merged**